packages/core/core/src/public/Environment.js

```diff
@@ -110,6 +137,10 @@ export default class Environment implements IEnvironment {
     return this.#environment.outputFormat;
   }
 
+  get sourceType(): ?SourceType {
```

Comment:
is this really a maybe type? whenever `createEnvironment()` is used it always has a SourceType